plenum/test/input_validation/fields_validation/test_version_field.py

```diff
@@ -1,44 +1,61 @@
 import pytest
 
+from plenum.common.version import DigitDotVersion
 from plenum.common.messages.fields import VersionField
 from plenum.config import VERSION_FIELD_LIMIT
 
-validator = VersionField(components_number=(2, 3,), max_length=VERSION_FIELD_LIMIT)
 
+class VersionTestCls(DigitDotVersion):
+    def __init__(self, version: str, **kwargs):
+        super().__init__(version, parts_num=(2, 3), **kwargs)
 
-def test_empty_version():
+
+max_length = 20
+legacy_validator = VersionField(components_number=(2, 3,), max_length=max_length)
+validator = VersionField(version_cls=VersionTestCls, max_length=max_length)
```

Comment:
But legacy validation also uses `DigitDotVersion`, so is it intentional that we validate by the same Version Validator?
Should we also parametrize using `SemVerReleaseVersion`?